02_first/supervised_pizzas.livemd

```diff
@@ -74,15 +74,15 @@ defmodule C2.LinearRegression do
   end
 
   def train(x, y, iterations, lr) when is_list(x) and is_list(y) do
-    Enum.reduce(0..iterations, 0, fn i, w ->
+    Enum.reduce_while(0..iterations, 0, fn i, w ->
```

Comment:
Not related to your changes, but while we are here. (adjusting the range was something else in my todo list that I always postponed 🙈)

```suggestion
    Enum.reduce_while(0..(iterations - 1), 0, fn i, w ->
```